code-challenges.js

```diff
@@ -23,12 +53,44 @@ var middleLetters1 = "hello"
 var middleLetters2 = "rhinoceros"
 // Expected output: “oc”
 
+// create a function that accepts a string
+const midLetters = (string) => {
+    // if the string is even
+    if (string.length % 2 === 0) {
+        // return the middle 2 letters
+        return string.charAt((string.length)/2 - 1) + string.charAt((string.length)/2) 
+ 
+    } else {
+        // else if the string is odd, find the middle letter using floor
+        // return the middle letter
+        return  string.charAt(Math.floor(string.length/2))
+    }
+}
 
+console.log(midLetters(middleLetters1))
+console.log(midLetters(middleLetters2))
 
 
 // --------------------4) Create a CLASS to get the area of a sphere. Create three spheres with different radi as test cases. Area of a sphere =  4πr^2 (four pi r squared)
 
+class Sphere {
+    constructor(radius){
+        this.radius = radius
+    }
+    area() {
+        return 4 * Math.PI* this.radius ** 2
```

Comment:
This is great. I'd also want to offer you a tip, take a look into Math.Pow. It might also do what you're looking for.